<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Tutorial</title>
    <!-- 设置网站logo -->
    <link rel="icon" href="../../img/sugarcane-logo.jpeg" type="image/x-icon">
    <link rel="stylesheet" href="../../css/reset.css">
    <link rel="stylesheet" href="../../css/test/basicPage.css">
    <link rel="stylesheet" href="../../css/test/tutorial.css">

    <!-- 引入字体库 -->
    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>

    <!-- 引入 lodash.js 用于 data.js 的数据深拷贝 -->
    <script src="https://cdn.jsdelivr.net/npm/jquery@3.7.1/dist/jquery.min.js"></script>
    <script src="https://cdn.jsdelivr.net/npm/echarts@5.5.0/dist/echarts.js"></script>
    <script src="https://cdn.jsdelivr.net/npm/lodash@4.17.21/lodash.min.js"></script>


</head>

<body>
    <div id="contact_overlay" class="contact_overlay"></div>
    <div id="contact_card" class="contact_card">
        <h2>Contact</h2>
        <p class="intro_text">Should there be any question, please let us know without hesitation!</p>
        <p class="content_text"><strong>Yanhan Shao (Web Design & Development):&nbsp;&nbsp;</strong><span
                class="email_address" title="click to copy">1740569155@qq.com</span>
        </p>
        <p class="content_text"><strong>Zhiyuan Zhao (Data Analyse):&nbsp;&nbsp;</strong><span class="email_address"
                title="click to copy">zhaozhiyuan@webmail.hzau.edu.cn</span></p>
        <p class="content_text"><strong>Professor Ling-Ling Chen:&nbsp;&nbsp;</strong><span class="email_address"
                title="click to copy">llchen@gxu.edu.cn</span></p>
        <p class="content_text"><strong>Professor Weibo Xie:&nbsp;&nbsp;</strong><span class="email_address"
                title="click to copy">weibo.xie@mail.hzau.edu.cn</span></p>
        <p class="content_text"><strong>Professor Lingqiang Wang:&nbsp;&nbsp;</strong><span class="email_address"
                title="click to copy">1335585196@qq.com</span></p>

        <p class="content_text"><strong>Project Source:</strong> <a href="https://github.com/shaoyanhan/GT42_Website"
                target="_blank">https://github.com/shaoyanhan/GT42_Websites</a></p>

        <p class="location_text"><strong>Location 1:</strong> &nbsp;&nbsp;State Key Laboratory for Conservation
            and
            Utilization of
            Subtropical Agro-bioresources, Guangxi University, Nanning
            530000, China
        </p>
        <p class="location_text"><strong>Location 2:</strong> &nbsp;&nbsp;National Key Laboratory of Crop
            Genetic
            Improvement, Hubei
            Hongshan Laboratory, Huazhong Agricultural University, Wuhan,
            430070, China
        </p>
    </div>

    <div class="center_container" style="overflow: none;">

        <!-- 返回顶部图标 -->
        <button class="scroll_to_top_button">
            <img src="../../img/scroll_to_top_icon.png" alt="scroll to top">
        </button>


        <!-- 顶部导航栏 -->
        <header class="nav_header">
            <div class="nav_container">
                <nav class="nav_bar">
                    <div class="logo_container">
                        <div class="logo_box">
                            <a href="./index.html"><img src="../../img/sugarcane-logo.jpeg" alt="logo"></a>

                        </div>
                        <div class="title_box">
                            <a href="./index.html">
                                <h1>SRNDB</h1>
                            </a>

                        </div>

                    </div>

                    <!-- 导航栏菜单 -->
                    <div class="menu_container">
                        <ul class="menu_links">
                            <li class="first_level_link"><a href="./index.html">Home</a></li>
                            <li class="first_level_link dropdown"><a>Data <span class="arrow">&#9660;</span></a>
                                <ul class="dropdown_content">
                                    <li class="second_level_link"><a href="./fullLengthTranscriptome.html"
                                            target="_blank">Full-length Transcriptome</a>
                                    </li>
                                    <li class="second_level_link"><a href="./haplotypeSNP.html"
                                            target="_blank">Haplotype &&
                                            SNP</a></li>
                                    <li class="second_level_link"><a href="./geneExpressionProfile.html"
                                            target="_blank">Gene
                                            Expression Profile</a></li>
                                    <li class="second_level_link"><a href="./singleCoExpressionNetwork.html"
                                            target="_blank">Single Co-expression
                                            Network</a></li>
                                    <li class="second_level_link"><a href="./hubCoExpressionNetwork.html"
                                            target="_blank">Hub
                                            Co-expression Network</a></li>
                                </ul>
                            </li>
                            <li class="first_level_link dropdown"><a>Tools<span class="arrow">&#9660;</span></a>
                                <ul class="dropdown_content">
                                    <li class="second_level_link"><a href="./blast.html" target="_blank">BLAST</a></li>
                                    <li class="second_level_link"><a href="#tool1">Annotation</a></li>
                                    <li class="second_level_link"><a href="#tool2">Transcript Factors</a></li>
                                </ul>
                            </li>
                            <li class="first_level_link"><a href="./download.html" target="_blank">Download</a></li>
                            <li class="first_level_link dropdown"><a>About<span class="arrow">&#9660;</span></a>
                                <ul class="dropdown_content">
                                    <li class="second_level_link"><a href="./tutorial.html" target="_blank">Tutorial</a>
                                    </li>
                                    <li class="second_level_link"><a class="contact_button">Contact</a></li>
                                </ul>
                            </li>
                        </ul>
                    </div>

                    <!-- 导航栏搜索框 -->
                    <div class="search_box_container">
                        <input type="text" placeholder="Search...">
                        <button type="submit" class="search_button"><span class="search_icon">&#128269;</span></button>

                        <div class="search_box_tooltip">
                            <span class="tooltip_text">Show the avaliable resources of specific ID been inputted.</span>

                            <div class="example_ids">
                                <span class="example_title">e.g.</span>
                                <span class="example_id">SGI000001.SO.01</span>
                            </div>

                            <button class="download_button">download full available ID list</button>
                        </div>
                    </div>

                    <button class="menu_toggle"><i class="iconfont icon-liebiao"></i></button>
                    <button class="menu_close"><i class="iconfont icon-31guanbi"></i></button>



                </nav>

                <!-- 响应式布局情况下的导航栏 -->
                <div class="expanded_nav">
                    <!-- 搜索框 -->
                    <div class="search_box_container">
                        <input type="text" placeholder="Search...">
                        <button type="submit" class="search_button"><span class="search_icon">&#128269;</span></button>

                        <div class="search_box_tooltip">
                            <span class="tooltip_text">Show the avaliable resources of specific ID been inputted.</span>

                            <div class="example_ids">
                                <span class="example_title">e.g.</span>
                                <span class="example_id">SGI000001.SO.01</span>
                            </div>

                            <button class="download_button">download full available ID list</button>
                        </div>
                    </div>
                    <!-- 手风琴导航栏菜单 -->
                    <div class="menu_container">
                        <ul class="menu_links">
                            <li class="first_level_link"><a href="./index.html">Home</a></li>
                            <li class="first_level_link accordion"><a>Data <span class="arrow">&#9660;</span></a>
                                <ul class="accordion_content">
                                    <li class="second_level_link"><a href="./fullLengthTranscriptome.html"
                                            target="_blank">Full-length Transcriptome</a>
                                    </li>
                                    <li class="second_level_link"><a href="./haplotypeSNP.html"
                                            target="_blank">Haplotype &&
                                            SNP</a></li>
                                    <li class="second_level_link"><a href="./geneExpressionProfile.html"
                                            target="_blank">Gene
                                            Expression Profile</a></li>
                                    <li class="second_level_link"><a href="./singleCoExpressionNetwork.html"
                                            target="_blank">Single Co-expression
                                            Network</a></li>
                                    <li class="second_level_link"><a href="./hubCoExpressionNetwork.html"
                                            target="_blank">Hub
                                            Co-expression Network</a></li>
                                </ul>
                            </li>
                            <li class="first_level_link accordion"><a>Tools<span class="arrow">&#9660;</span></a>
                                <ul class="accordion_content">
                                    <li class="second_level_link"><a href="./blast.html" target="_blank">BLAST</a></li>
                                    <li class="second_level_link"><a href="#tool1">Annotation</a></li>
                                    <li class="second_level_link"><a href="#tool2">Transcript Factors</a></li>
                                </ul>
                            </li>
                            <li class="first_level_link"><a href="./download.html" target="_blank">Download</a></li>
                            <li class="first_level_link accordion"><a>About<span class="arrow">&#9660;</span></a>
                                <ul class="accordion_content">
                                    <li class="second_level_link"><a href="./tutorial.html" target="_blank">Tutorial</a>
                                    </li>
                                    <li class="second_level_link"><a class="contact_button">Contact</a></li>
                                </ul>
                            </li>
                        </ul>
                    </div>


                </div>
            </div>



        </header>

        <div class="content_container">


            <!-- 内容部分 -->
            <div class="content">
                <h2 id="homepage">
                    <div class="title_container">
                        <a href="#homepage" class="title_link">Homepage</a>
                        <img class="title_link_icon" src="../../img/link_icon.png" alt="link">
                    </div>
                </h2>
                <p class="h2_p">This page provides an overview of the research and the database content, along with a
                    user-friendly and intuitive guide
                    to the features.</p>

                <h3 id="abstract_citation">
                    <div class="title_container">
                        <a href="#abstract_citation" class="title_link">Abstract & Citation</a>
                        <img class="title_link_icon" src="../../img/link_icon.png" alt="link">
                    </div>
                </h3>
                <ul class="h3_p">
                    <li>Here you can view the detailed article summary and find citation information.</li>
                </ul>

                <h3 id="statistics">
                    <div class="title_container">
                        <a href="#statistics" class="title_link">Statistics</a>
                        <img class="title_link_icon" src="../../img/link_icon.png" alt="link">
                    </div>
                </h3>
                <ul class="h3_p">
                    <li>Here, a brief statistical overview of the database's data is provided, with all charts
                        being
                        interactive.
                        <div class="image_container">
                            <img src="../../img/tutorial1.png" alt="Image description" class="clickable_image">
                            <div class="image_caption">Fig 1. The statistics section</div>
                        </div>
                        <ul>
                            <li><strong>Full-length Transcriptome (Fig 1a): </strong>Describes the scale of the
                                transcriptome
                                database
                                constructed in this study and enumerates the number of full-length
                                transcripts corresponding to different chimeric genes.</li>
                            <li><strong>Variation (Fig 1b): </strong>Displays information on the number of SNPs in the
                                transcriptome
                                database and provides statistics on the proportions of
                                different types of SNPs.</li>
                            <li><strong>Gene Expression Profile (Fig 1c): </strong>Details the sample types and
                                quantities used
                                in this
                                study, and presents the average TPM values for each sample at four
                                resolutions (orthologous/mosaic, xenologous, gene/haplotype, transcript).</li>
                            <li><strong>Co-expression Network (Fig 1d): </strong>Shows the network scale at three
                                resolutions
                                (mosaic,
                                xenologous, gene/haplotype) and provides a count of edge weights
                                in four ranges: (1, 2), [2, 3), [3, 4), and [4, 11).</li>
                        </ul>
                    </li>

                </ul>

                <h3 id="resources_list">
                    <div class="title_container">
                        <a href="#resources_list" class="title_link">Resources List</a>
                        <img class="title_link_icon" src="../../img/link_icon.png" alt="link">
                    </div>
                </h3>
                <ul class="h3_p">
                    <li>Here, the appearances of five database pages (Fig 2a. Full-length Transcriptome, Haplotype &
                        SNP, Gene
                        Expression Profile,
                        Single Co-expression Network, Hub Co-expression Network) and three online tool pages
                        (Blast,
                        Annotation, Transcript
                        Factors) are dynamically displayed. You can click on the corresponding buttons to access these
                        pages and explore them
                        further.
                    </li>
                    <div class="image_container">
                        <img src="../../img/tutorial2.png" alt="Image description" class="clickable_image">
                        <div class="image_caption">Fig 2. The resources section</div>
                    </div>

                </ul>

                <h3 id="news_list">
                    <div class="title_container">
                        <a href="#news_list" class="title_link">News List</a>
                        <img class="title_link_icon" src="../../img/link_icon.png" alt="link">
                    </div>
                </h3>
                <ul class="h3_p">
                    <li>This section will showcase important updates on website development and data content.
                    </li>
                </ul>


                <h2 id="navigation_bar">
                    <div class="title_container">
                        <a href="#navigation_bar" class="title_link">Navigation Bar</a>
                        <img class="title_link_icon" src="../../img/link_icon.png" alt="link">
                    </div>
                </h2>

                <h3 id="responsive_layout">
                    <div class="title_container">
                        <a href="#responsive_layout" class="title_link">Responsive Layout</a>
                        <img class="title_link_icon" src="../../img/link_icon.png" alt="link">
                    </div>
                </h3>
                <ul class="h3_p">
                    <li>The navigation bar will remain fixed at the top of the screen on all pages and
                        dynamically
                        adjust its length according
                        to the user's screen size (Fig 3a). When the screen is narrower, the options and search box will
                        be
                        hidden, and users will need
                        to click the menu button to open the navigation menu (Fig 3b).
                    </li>
                    <div class="image_container">
                        <img src="../../img/tutorial3.png" alt="Image description" class="clickable_image">
                        <div class="image_caption">Fig 3. Responsive navigation bar</div>
                    </div>

                </ul>

                <h3 id="global_search">
                    <div class="title_container">
                        <a href="#global_search" class="title_link">Global Search</a>
                        <img class="title_link_icon" src="../../img/link_icon.png" alt="link">
                    </div>
                </h3>
                <ul class="h3_p">
                    <li>Users can enter a database ID into the search box in the navigation bar to perform a
                        global
                        search (Fig 4a). This will generate a
                        new navigation page directing users to the resource page associated with that ID (Fig 4c). If
                        the ID is
                        invalid for a particular
                        resource page, the resource button will become non-clickable (Fig 4d). A new ID list has been
                        created for
                        the full-length
                        transcriptome database developed for this research. Users can hover over the search box to
                        download this list (Fig 4b), which
                        also indicates the precision levels (orthologous/mosaic, xenologous, gene/haplotype, transcript)
                        for different IDs.</li>
                    <div class="image_container">
                        <img src="../../img/tutorial4.png" alt="Image description" class="clickable_image">
                        <div class="image_caption">Fig 4. Global search box</div>
                    </div>
                </ul>

                <h2 id="graph_summary">
                    <div class="title_container">
                        <a href="#graph_summary" class="title_link">Graph Summary</a>
                        <img class="title_link_icon" src="../../img/link_icon.png" alt="link">
                    </div>
                </h2>
                <div class="image_container">
                    <img src="../../img/workflow.jpg" alt="Image description" class="clickable_image">
                    <div class="image_caption">Fig 5. The workflow of dataset generation</div>
                </div>
                <ul class="h3_p">
                    <li>
                        Our research data can be summarized with Fig 5. We conducted RNA-seq and Iso-seq
                        sequencing on 23 tissues across
                        five developmental stages of GT42 sugarcane. The RNA-seq results were used to correct the
                        Full-Length Non Chimeric
                        (FLNC) sequences. Using the haploid <span class="italic">Erianthus rufipilus</span> genome as a
                        reference, we successfully
                        assembled a high-quality
                        mosaic gene dataset. Next, by performing SNP calling on the FLNC sequences, we clustered the
                        RNA-seq and Iso-seq reads
                        based on SNP loci. The clustered reads were then reassembled to obtain a haplotype-resolved gene
                        dataset. Finally, the
                        FLNC sequences were assigned to different haplotype genes, resulting in the Full-length
                        Transcriptome dataset.
                    </li>
                </ul>


                <h2 id="data_resources">
                    <div class="title_container">
                        <a href="#data_resources" class="title_link">Data Resources</a>
                        <img class="title_link_icon" src="../../img/link_icon.png" alt="link">
                    </div>
                </h2>
                <p class="h2_p">To provide a comprehensive view of the database data, we have created five data
                    visualization pages with a clean and
                    elegant UI. You can explore the database content through a range of interactive features.</p>

                <h3 id="full_length_transcriptome">
                    <div class="title_container">
                        <a href="#full_length_transcriptome" class="title_link">Full-length Transcriptome</a>
                        <img class="title_link_icon" src="../../img/link_icon.png" alt="link">
                    </div>
                </h3>
                <ul class="h3_p">
                    <li>The <strong>"Haplotype Profile"</strong> bar chart displays all haplotype-resolution genes
                        (referred to as
                        haplotype genes) derived from
                        a mosaic gene. The top blue bar represents the mosaic gene (Fig 6a), while all bars below it
                        represent
                        the haplotype genes
                        extracted from this mosaic (Fig 6b). The bars are aligned at the start, but their positions do
                        not
                        indicate the actual spatial
                        alignment of the haplotype genes with the mosaic gene. The length of each bar represents the
                        length of the gene. There
                        are five colors used for the bars: blue for the mosaic gene, cyan-green for haplotype genes
                        highly homologous to <span class="italic">S.
                            officinarum</span>, dark green for haplotype genes highly homologous to <span
                            class="italic">S. spontaneum</span>, pink for highly
                        conserved haplotype
                        genes, and brown for haplotype genes with unknown homology.</li>
                    <div class="image_container">
                        <img src="../../img/tutorial6.png" alt="Image description" class="clickable_image">
                        <div class="image_caption">Fig 6. The haplotype and full-length transcript plot</div>
                    </div>
                    <li>The <strong>"Transcripts Profile of Haplotype"</strong> bar chart represents all alternative
                        splicing events for a mosaic gene or
                        haplotype gene (Fig 6c). In the chart, exons of all transcripts are shown with light blue bars
                        (Fig 6d), while
                        introns are represented by
                        narrower yellow bars (Fig 6e). The position of each transcript is aligned with the actual
                        position
                        of its corresponding
                        gene. When a mosaic ID is searched in the “search ID” box, the chart will display all
                        alternative splicing events for
                        all haplotype genes associated with that mosaic ID. In this case, the chart will show all
                        haplotype genes and their
                        corresponding alternative splicing events, with the haplotype genes colored the same as in the
                        <strong>"Haplotype Profile"</strong>
                        bar chart.
                    </li>
                    <li>The page offers a range of interactive features. In the <strong>"search ID"</strong> input box,
                        users can
                        enter an ID to display its
                        visualization results (Fig 7a). You can check the supported ID formats and types via the help
                        button, or
                        directly
                        download the ID list to
                        view IDs for the four resolution levels (orthologous/mosaic, xenologous, gene/haplotype,
                        transcript). Additionally, you
                        can browse the results sequentially for mosaic IDs by clicking the <strong>Previous</strong> and
                        <strong>Next</strong>
                        buttons, though this
                        functionality is only available for mosaic IDs. Furthermore, the search box will recognize the
                        ID entered by the user
                        and highlight the corresponding bar in the bar chart. Both bar charts include a <strong>"Result
                            Details"</strong> card on the right side (Fig 7b, c), which
                        displays detailed information about the selected bar and
                        updates in real-time as users click on different bars. Below the bar charts, a hidden section
                        contains a table (haplotype table / transcript table) with all
                        the information from the charts and provides a download option.
                    </li>
                    <div class="image_container">
                        <img src="../../img/tutorial7.png" alt="Image description" class="clickable_image">
                        <div class="image_caption">Fig 7. The ID search tool and result details card</div>
                    </div>
                    <li>You can hover over different elements in the image to view detailed information about each
                        element (Fig 8a, b). Additionally, you
                        can adjust the image range using the zoom tools located on the right and bottom of the image
                        (Fig 8d).
                        The functionality buttons
                        in the top right corner allow for image area selection and image downloading. When you click on
                        different bars in the
                        <strong>"Haplotype Profile"</strong> bar chart, the alternative splicing types in
                        <strong>"Transcripts Profile of
                            Haplotype"</strong> bar chart, the
                        <strong>"Result Details"</strong> card, and the information table will update in real-time. If
                        the table contains many columns, you can use
                        the horizontal scrollbar at the bottom of the table to view additional columns (Fig 8b).
                    </li>
                    <div class="image_container">
                        <img src="../../img/tutorial8.png" alt="Image description" class="clickable_image">
                        <div class="image_caption">Fig 8. The interactive components</div>
                    </div>
                </ul>

                <h3 id="haplotype_SNP">
                    <div class="title_container">
                        <a href="#haplotype_SNP" class="title_link">Haplotype & SNP</a>
                        <img class="title_link_icon" src="../../img/link_icon.png" alt="link">
                    </div>
                </h3>
                <ul class="h3_p">
                    <li>The <strong>"Haplotype and SNP Profile"</strong> bar chart displays the SNP positions and types
                        for each haplotype gene corresponding
                        to a mosaic gene. The color scheme for the bars is consistent with the definitions in the
                        <strong>Full-length Transcriptome</strong>
                        page: blue for the mosaic gene, cyan-green for haplotype genes
                        highly homologous to <span class="italic">S.
                            officinarum</span>, dark green for haplotype genes highly homologous to <span
                            class="italic">S. spontaneum</span>,
                        pink for highly
                        conserved haplotype
                        genes, and brown for haplotype genes with unknown homology. SNP sites are shown in five colors:
                        blue for A, red for C,
                        yellow for G, green for T, and black for sites without exon reads evidence.
                    </li>
                    <li>We performed <strong>SNP calling</strong> on both <strong>ISO-Seq</strong> and
                        <strong>RNA-Seq</strong> results and clustered reads that remapped to the mosaic gene based on
                        SNP information. Reads within the same cluster were reassembled, allowing us to map SNP sites to
                        each haplotype gene.
                        Consequently, we can associate SNP sites on the mosaic gene with haplotype genes. However,
                        <strong>the
                            SNP position number correspond
                            to the mosaic gene's location, not the haplotype gene's location. To represent the
                            distribution
                            of all SNP sites on a mosaic gene
                            across its haplotype
                            genes, we set the length of all haplotype gene bars to match the length of the mosaic
                            gene.</strong>
                        Therefore, each SNP marked
                        on a haplotype gene indicates the type of SNP at that position in the mosaic gene. For example,
                        SNP position 17 for
                        GT42G000001 is of type A/C because the reads aligned to this mosaic gene at position 17 may show
                        either A or C. Thus, through
                        clustering and reassembly of these reads, we can determine whether the SNP at position 17 for
                        each haplotype gene is A
                        or C.
                    </li>

                    <li>In the <strong>"Haplotype and SNP Profile"</strong> bar chart, you can hover over image elements
                        to view detailed
                        information about
                        specific mosaic genes, haplotype genes, and SNPs (Fig 9a-c). You can adjust the image range
                        using the zoom
                        tools on the right and
                        bottom of the image and use the functionality buttons in the top right corner to select image
                        areas and download the
                        image (Fig 9d). The results detail card on the right side updates when you click on image
                        elements. Note
                        that when you click on a
                        mosaic gene bar, the green <strong>"haplotype SNP"</strong> table in the <strong>"Result
                            Details"</strong> will display all SNP
                        sites and types on that
                        mosaic gene (Fig 9e). When you click on a haplotype gene bar, the table will show the specific
                        SNP types
                        for that haplotype gene (Fig 9f).
                        Clicking on an SNP site will display the number of ISO-Seq and RNA-Seq read evidences for that
                        SNP (Fig 9g), and the green table
                        will show the specific type of that SNP site for each haplotype gene (Fig 9h).</li>
                    <div class="image_container">
                        <img src="../../img/tutorial9.png" alt="Image description" class="clickable_image">
                        <div class="image_caption">Fig 9. The interactive components and SNP table</div>
                    </div>
                    <li>The <strong>"SNP with xxx Evidence"</strong> bar chart displays the ISO-Seq and RNA-Seq read
                        evidence statistics for all SNP sites on
                        each mosaic gene. The top subplot shows SNP sites supported by both ISO-Seq and RNA-Seq reads.
                        The middle subplot
                        represents SNP sites supported only by ISO-Seq reads, while the bottom subplot displays SNP
                        sites supported only by
                        RNA-Seq reads. You can hover over specific positions to view the SNP types associated with those
                        sites. The three
                        subplots are designed to be synchronized, allowing you to control the zoom and pan of all three
                        subplots simultaneously
                        by interacting with one of them.</li>
                    <li>Detailed data corresponding to both bar charts can be viewed and downloaded in the hidden
                        section below the image. If
                        the table's width is extensive, you can use the horizontal scrollbar at the bottom of the table
                        to view additional
                        columns.</li>
                    <li>The <strong>"search ID"</strong> box on this page only supports searching with mosaic IDs.
                        However, you still can
                        enter
                        other types of IDs, but
                        they will be converted to mosaic IDs for the search.</li>
                </ul>

                <h3 id="gene_expression_profile">
                    <div class="title_container">
                        <a href="#gene_expression_profile" class="title_link">Gene Expression Profile</a>
                        <img class="title_link_icon" src="../../img/link_icon.png" alt="link">
                    </div>
                </h3>
                <ul class="h3_p">
                    <li>
                        The <strong>"development stages"</strong> sunburst chart illustrates all sample information from
                        the study (Fig 10). We
                        collected <strong>23 tissues</strong> from
                        GT42 sugarcane at <strong>five developmental stages</strong> , with each tissue
                        <strong>sequenced in triplicate</strong>. The
                        samples in the sunburst chart
                        are arranged in a clockwise order according to developmental stages and are color-coded by
                        stage. The innermost ring
                        represents developmental stages, the middle ring represents tissue numbers, and the outermost
                        ring represents specific
                        sample numbers. You can hover over or click on specific segments to view the hierarchical
                        structure of specific developmental
                        stages.
                    </li>
                    <div class="image_container">
                        <img src="../../img/development_stage.gif" alt="Image description" class="clickable_image">
                        <div class="image_caption">Fig 10. The interactive development stage plot</div>
                    </div>
                    <li>
                        The <strong>"ID Tree"</strong> dendrogram illustrates the hierarchical structure of IDs across
                        four levels of resolution (Fig 11). From left to
                        right, the levels are represents for mosaic/orthologous ID, xenologous ID, haplotype/gene ID,
                        and transcript ID. Since a mosaic gene
                        may contain a large number of transcripts, transcript IDs are collapsed by default. However, you
                        can click on each
                        ID to expand or collapse the nodes as needed.
                    </li>
                    <div class="image_container">
                        <img src="../../img/id_tree.gif" alt="Image description" class="clickable_image">
                        <div class="image_caption">Fig 11. The interactive ID tree plot</div>
                    </div>
                    <li>
                        The page features four heatmap areas representing TPM levels at four different resolutions
                        (Fig 12). The
                        x-axis denotes tissue
                        numbers (corresponding to the outermost ring in the <strong>"development stages"</strong>
                        sunburst chart), while the y-axis denotes IDs
                        (matching the hierarchical structure in the <strong>"ID Tree"</strong> dendrogram). You can
                        hover over specific heatmap points to see
                        detailed expression data for the sample. Each heatmap has a horizontal color bar in the
                        top-right corner for adjusting
                        the color range of the heatmap data, and you can also use it to view data information by
                        hovering
                        over specific values. Additionally, a
                        color bar tool is available for changing the overall heatmap color scheme by selecting different
                        color blocks. Detailed data can
                        also be viewed and downloaded in tabular format from the hidden section below each heatmap
                        image.
                    </li>
                    <div class="image_container">
                        <img src="../../img/tutorial_heatmap.gif" alt="Image description" class="clickable_image">
                        <div class="image_caption">Fig 12. The interactive heatmap section</div>
                    </div>
                </ul>

                <h3 id="single_co_expression_network">
                    <div class="title_container">
                        <a href="#single_co_expression_network" class="title_link">Single Co-expression Network</a>
                        <img class="title_link_icon" src="../../img/link_icon.png" alt="link">
                    </div>
                </h3>
                <ul class="h3_p">
                    <li>
                        The <strong>"Single Co-expression Network"</strong> is a single-core network that primarily
                        displays all adjacent nodes and edges
                        connected to a central node. If there are additional edges connecting between the adjacent
                        nodes, those
                        connections are also
                        displayed within the single-core network. Thus, the number of network layers follows the
                        formula: <strong>1 ≤ number of layers < 2</strong>. The node colors in the graph are used only
                                to distinguish
                                between different nodes, while the edges share the same color as their source node. The
                                node
                                size is calculated using the formula: <strong>Symbol Size&nbsp;=&nbsp;math.log(
                                    in_degree +
                                    out_degree + 1 )</strong>, where log refers to the natural logarithm. The width of
                                the edges
                                represents the weight of the connection, and all the edges shown in the single-core
                                network
                                having a threshold of <strong>weight ≥ 1</strong>. </li>
                    <li>
                        You can zoom the image area using your mouse scroll wheel, and you can freely drag within the
                        image to
                        roaming (Fig 13). To help you explore the adjacent nodes and edges of a specific node, you can
                        hover over
                        a particular node or
                        edge to view its properties. When hovering over a node, all adjacent nodes and edges connected
                        to it in the current network will be
                        highlighted. Similarly, hovering over an edge will highlight its source and target nodes.
                        Additionally, you can click on a node or
                        edge to display detailed information in the result details card. If you wish to view detailed
                        information about all
                        nodes and edges in the current network, you can find or download a complete table from the
                        hidden section below the
                        image.
                    </li>
                    <div class="image_container">
                        <img src="../../img/tutorial_network1.gif" alt="Image description" class="clickable_image">
                        <div class="image_caption">Fig 13. The interactive core network</div>
                    </div>
                    <li>
                        We have assigned <strong>hyperlink attributes</strong> to all node IDs displayed in the results
                        detail card (Fig
                        14). You
                        can directly click on
                        any ID, which will then render the core network corresponding to the clicked ID. This allows you
                        to explore and navigate
                        through the entire co-expression network using the single-core network view, effectively
                        enabling seamless transitions
                        between different nodes.
                    </li>
                    <div class="image_container">
                        <img src="../../img/tutorial_network2.gif" alt="Image description" class="clickable_image">
                        <div class="image_caption">Fig 14. The infinite exploration feature</div>
                    </div>
                    <li>
                        You can switch the resolution of the current network using the <strong>resolution selection
                            bar</strong> located at the top left of the
                        image. We currently offer three network resolution levels: mosaic/orthologous, xenologous, and
                        gene/haplotype (Fig 15). After switching,
                        the core network of the first homologous ID under the selected resolution will be displayed by
                        default. If there are
                        multiple homologous IDs for the selected resolution, such as xenologous ID and gene/haplotype
                        ID, you can choose and
                        switch between them using the <strong>"Select Homologous ID"</strong> dropdown menu to render
                        the core network for that specific ID.
                        Please note that <strong>the total degree of a core network may affect the rendering time and
                            display
                            smoothness</strong>, but once the
                        network iteration is complete, it will return to normal smoothness.
                    </li>
                    <div class="image_container">
                        <img src="../../img/tutorial_network3.gif" alt="Image description" class="clickable_image">
                        <div class="image_caption">Fig 15. The resolution switching tools</div>
                    </div>
                </ul>

                <h3 id="hub_co_expression_network">
                    <div class="title_container">
                        <a href="#hub_co_expression_network" class="title_link">Hub Co-expression Network</a>
                        <img class="title_link_icon" src="../../img/link_icon.png" alt="link">
                    </div>
                </h3>
                <ul class="h3_p">
                    <li>
                        To illustrate the overall regulatory situation of the nodes, we have filtered out hub nodes
                        under three resolution
                        levels from the original data for display. The <strong>selection criteria</strong> for hub nodes
                        are as follows:
                        Mosaic: <strong>symbol size ≥ 2.5, weight ≥ 2</strong>;
                        Xenologous: <strong>symbol size ≥ 2.5, weight ≥ 2</strong>;
                        Gene: <strong>symbol size ≥ 4.2, weight ≥ 2</strong>.

                        The final <strong>number of nodes and edges</strong> obtained from each resolution level is:
                        Mosaic: <strong>nodes = 1,474, edges = 44,925</strong>;
                        Xenologous: <strong>nodes = 2,475, edges = 67,990</strong>;
                        Gene: <strong>nodes = 3,461, edges = 150,127</strong>.
                    </li>
                    <li>
                        In the hub network, the size of each node is calculated using the formula: <strong>Symbol Size =
                            math.log( in_degree + out_degree
                            + 1 )</strong>, where log refers to the natural logarithm. Since the hub network contains a
                        large number
                        of edges with higher
                        weights, to enhance image clarity and improve interactive response speed, we <strong>no longer
                            use edge
                            width to represent its
                            weight</strong>. Additionally, the focus effect when hovering with the mouse is now limited
                        to the
                        currently selected node and
                        its connected edges.
                    </li>
                    <li>
                        Since the hub network only displays hub nodes, when you click on a node, the result details card
                        will not show all of
                        its adjacent nodes from the original network. However, <strong>you can still click on the ID in
                            the
                            result details card to
                            navigate to the Single Co-expression Network page</strong> (Fig 16), where you can visualize
                        all adjacent
                        nodes
                        and edges of the hub node.
                        Alternatively, you can directly view or download the information table from the hidden section
                        below the image, which
                        contains the original records of all nodes and edges under the current precision level
                        <strong>(edge
                            weight ≥ 1)</strong>.
                    </li>
                    <div class="image_container">
                        <img src="../../img/tutorial_network4.gif" alt="Image description" class="clickable_image">
                        <div class="image_caption">Fig 16. Link from hub network to single network</div>
                    </div>
                    <li>
                        Please note that due to the large number of nodes and edges in the hub network, there may be
                        some lag when the network
                        graph is initially loaded. However, we have implemented caching optimizations to ensure that
                        each of the three
                        resolution networks will only be loaded once during the current session. After the network
                        iteration is complete, if
                        your computer has sufficient memory, your browser will be allocated enough resources, allowing
                        the webpage to run
                        smoothly without significant impact on performance.
                    </li>
                </ul>

                <h2 id="tools">
                    <div class="title_container">
                        <a href="#tools" class="title_link">Tools</a>
                        <img class="title_link_icon" src="../../img/link_icon.png" alt="link">
                    </div>
                </h2>

                <h3 id="blast">
                    <div class="title_container">
                        <a href="#blast" class="title_link">BLAST</a>
                        <img class="title_link_icon" src="../../img/link_icon.png" alt="link">
                    </div>
                </h3>
                <ul class="h3_p">
                    <li>
                        This research has ultimately created four sequence datasets: <strong>nucleotide sequences
                            dataset of
                            mosaic genes, haplotype genes, full-length transcripts, and protein sequences dataset of
                            full-length transcripts</strong>. To enable users to better
                        explore these datasets, we have developed a powerful and user-friendly BLAST front-end
                        visualization platform. The
                        back-end of this platform is based on <strong>NCBI’s BLAST 2.16.0+</strong> localized tool, and
                        we have
                        pre-integrated these four
                        datasets into the front-end search
                        interface. Compared to most complex BLAST tools, we have significantly simplified the
                        traditional BLAST functionalities
                        and created a range of user-friendly interactive features with a clean and streamlined UI.
                    </li>
                    <li> <strong>Search steps:</strong>
                        <ul>
                            <li>Step 1: You can either copy and paste some query sequences in FASTA format into the
                                input box or
                                drag and drop a file containing the query
                                sequence into the input box for automatic filling. Additionally, you could also just
                                click the
                                "example" button to automatically
                                retrieve two nucleotide or protein query sequences (Fig 17).</li>
                            <div class="image_container">
                                <img src="../../img/tutorial_blast1.gif" alt="Image description"
                                    class="clickable_image">
                                <div class="image_caption">Fig 17. Filling query box</div>
                            </div>
                            <li>Step 2: Once the query sequence is detected to be valid, the checkboxes of "database"
                                will automatically become active (Fig 18). You will need to
                                select one or more database types for the search. You can only choose either nucleotide
                                or protein databases at the same time, but you can select multiple databases of the same
                                type at one time, which allows for more comprehensive BLAST results.</li>
                            <li>Step 3: Once you have completed entering the query sequence and selecting the database,
                                the system will automatically determine
                                which types of BLAST functions are available based on your query sequence type and
                                database combination, and then enabling the
                                corresponding buttons (Fig 18). Additionally, you can hover over each BLAST function
                                button to
                                view a brief description of its
                                algorithm in advance.</li>
                            <div class="image_container">
                                <img src="../../img/tutorial_blast2.gif" alt="Image description"
                                    class="clickable_image">
                                <div class="image_caption">Fig 18. Selecting database and BLAST function</div>
                            </div>
                            <li>Step 4: If the user input is valid and the backend software runs successfully, the
                                server will process your task and return the
                                results. After the results are loaded, a BLAST result box will appear below the
                                submission form. <strong></strong>If you submitted
                                multiple query sequences, there will be multiple result sets, but the BLAST result box
                                will only display the
                                visualization for one query at a time. To view the visualization for other query
                                sequences, you can switch the query ID
                                using the "Select Result for"</strong> dropdown menu (Fig 19a). You can also
                                directly
                                download the result files
                                for the current query in three
                                formats from the "Download" dropdown on the right (Fig 19b), which includes:

                                <strong>Pairwise Alignment</strong>: The most common BLAST result, showing alignment
                                score and
                                pairwise comparison in text format.
                                <strong>Detailed Value Table</strong>: A detailed table with comments displaying
                                alignment score data.
                                <strong>Comprehensive JSON</strong>: Contains detailed search parameters and result
                                data, suitable
                                for custom analysis.
                            </li>
                            <div class="image_container">
                                <img src="../../img/tutorial_blast3.png" alt="Image description"
                                    class="clickable_image">
                                <div class="image_caption">Fig 19. Switching results between query and download result
                                    files</div>
                            </div>
                        </ul>
                    </li>
                    <li><strong>All Hits Bar Plot:</strong>
                        <ul>
                            <li>
                                This chart visualizes the results of Detailed Value Table, showing <strong>all matching
                                    intervals</strong>. The x-axis represents the length
                                of the query sequence, and the y-axis represents the subject/target sequence ID. Since
                                all HSPs
                                (High-scoring Segment Pairs / High Scoring Pairs)
                                within a single Hit share the same subject ID, <strong>only the first HSP's ID is
                                    displayed for
                                    each Hit (Fig 20a)</strong>.
                            </li>
                            <div class="image_container">
                                <img src="../../img/tutorial_blast4.png" alt="Image description"
                                    class="clickable_image">
                                <div class="image_caption">Fig 20. All Hits Bar Plot</div>
                            </div>
                            <li>
                                Each bar in the chart indicates a matching region between a section of the query
                                sequence and a corresponding section of
                                the subject sequence with the specific ID on the y-axis. The direction of the bar
                                indicates the
                                alignment direction of the <strong>query
                                    sequence</strong>. You can hover over a bar to view detailed matching descriptions
                                or use your
                                mouse scroll wheel or the zoom
                                tool on the right side of the image to zoom in or out (Fig 20b, c).
                            </li>
                            <li>
                                In the result information, <strong>frames</strong> refer to ORFs (Open Reading Frames).
                                Since blastn and
                                blastp do not involve ORF
                                translation, so the number 1 represents a sequence starting from the beginning. For
                                nucleotide sequences, both strands
                                (positive:1 and negative:-1) must be considered, whereas protein sequences do not have
                                this
                                concept. In blastx, tblastx, and
                                tblastn, the sequence is translated into six reading frames. However, protein sequences
                                do not have the concept of reading frames, so
                                in BLAST functions with ORF translation, protein sequences use 0 to represent a protein
                                sequence starting from the
                                beginning, while nucleotide sequences use ±1, ±2, and ±3 to represent different ORF
                                start points and alignment directions.
                            </li>
                            <li>
                                You can click on a specific bar to jump to the detailed pairwise sequence alignment
                                visualization result section.
                            </li>
                        </ul>
                    </li>
                    <li>
                        <strong>All Hits Table:</strong> This table displays the data from the <strong>Detailed Value
                            Table</strong>. To
                        make the
                        table more concise, we have abbreviated the
                        column headers. You can hover over a specific header or table data item to view the full title.
                        Additionally, all
                        Subject IDs in the table are hyperlinked, allowing you to click on them to open the
                        <strong>global
                            search</strong> page, which will
                        display resource page redirect links related to that specific ID (Fig 21).
                    </li>
                    <div class="image_container">
                        <img src="../../img/tutorial_blast5.gif" alt="Image description" class="clickable_image">
                        <div class="image_caption">Fig 21. All Hits Table</div>
                    </div>
                    <li><strong>All Hits Alignment Details:</strong>
                        <ul>
                            <li>
                                This area displays the pairwise alignment visualization results for all Hits. Each Hit
                                is presented in a result section
                                consisting of four distinct panels.
                            </li>
                            <li>
                                <strong>Result file download:</strong> In the <strong>"Download"</strong> dropdown of
                                each result
                                section (Fig 22a), you can
                                download six types of results:
                                <strong>Area Picture:</strong> Download the SVG image of the current sequence region
                                alignment.
                                <strong>Alignment Sequence Picture:</strong> Download the HTML text result of the
                                current HSP sequence
                                alignment result.
                                <strong>Complete Subject Sequence:</strong> Download the complete nucleotide or protein
                                sequence of the
                                subject/target subject.
                                <strong>Aligned Subject Sequence:</strong> Download the nucleotide or protein sequences
                                of the subject/target
                                corresponding to
                                all HSP regions for the current
                                Hit, with HSP numbers indicated in the file.
                                <strong>Pairwise Alignment Result:</strong> Download the pairwise sequence alignment
                                text results for all
                                HSP regions in the current Hit.
                                <strong>Detailed Value Table:</strong> Download the numerical result table for all HSPs
                                of the current
                                subject/target sequence.
                            </li>
                            <div class="image_container">
                                <img src="../../img/tutorial_blast6.png" alt="Image description"
                                    class="clickable_image">
                                <div class="image_caption">Fig 22. Pairwise alignment visualization section</div>
                            </div>
                            <li>
                                <strong>Jumping buttons:</strong> You can use the four navigation buttons on
                                the right to jump
                                between results within the All
                                Hits Alignment Details section (Fig 22b):

                                <strong>Top:</strong> Return to the All Hits Bar Plot.
                                <strong>Previous:</strong> Go to the previous hit result.
                                <strong>Next:</strong> Go to the next hit result.
                                <strong>Bottom:</strong> Jump to the last hit result.
                                To improve webpage performance, the All Hits Alignment Details result section uses lazy
                                loading technology. Only 10 hit
                                result panels will be loaded at a time. When you scroll to the bottom or click the
                                "bottom"
                                button, the next or all batch of result panels will
                                be loaded. However, if the number of hits exceeds 100, loading all batch
                                result panels at once can significantly affect
                                the page's interactivity.
                            </li>
                            <li>
                                <strong>Result Details Card:</strong> This card displays the detailed numerical results
                                of the
                                selected HSP within
                                a Hit. You can click the
                                <strong>"Previous"</strong> and <strong>"Next"</strong> buttons in the HSP Index section
                                to switch between HSP regions (Fig 22c).
                                The HSP focus area in the matching
                                region chart of left and the sequence alignment text result below will also update in
                                real-time.
                            </li>
                            <li>
                                <strong>Matching Region Chart:</strong> This chart shows the alignment between the
                                subject sequence and
                                the query sequence. All HSPs that are
                                visualized on this chart—forward matches are shown as quadrilateral regions, while
                                reverse matches are displayed as
                                funnel-shaped regions. You can zoom in and out of x-axises by scrolling the
                                mouse wheel on the top or bottom half area of the plot area, and you can also move the
                                x-axises by
                                dragging them left or right (Fig 23). When you click on a specific HSP, it will be
                                highlighted,
                                and the results detail card on
                                the left, as well as the sequence alignment text below, will update in real-time.
                                However, if a Hit contains many HSP
                                regions, it may be more convenient to use the "Previous" and "Next" buttons in the HSP
                                Index section of the result
                                details card to switch between HSPs.
                            </li>
                            <div class="image_container">
                                <img src="../../img/tutorial_blast7.gif" alt="Image description"
                                    class="clickable_image">
                                <div class="image_caption">Fig 23. Matching region chart</div>
                            </div>
                            <li>
                                <strong>Sequence Alignment Text Result:</strong> We've added vibrant color effects to
                                the otherwise
                                monotonous nucleotide/peptide alignment
                                results (Fig 22d). The numbers at both ends of the sequence represent the position of
                                the first
                                and last base between this region. The sequence
                                numbering starts at 1, and since gaps are not counted in the total sequence length, the
                                difference between the numbers
                                of the first and last base might be smaller than the actual number of elements
                                displayed.
                            </li>
                        </ul>
                    </li>
                </ul>

                <h3 id="annotation">
                    <div class="title_container">
                        <a href="#annotation" class="title_link">Annotation</a>
                        <img class="title_link_icon" src="../../img/link_icon.png" alt="link">
                    </div>
                </h3>

                <h3 id="transcript_factors">
                    <div class="title_container">
                        <a href="#transcript_factors" class="title_link">Transcript Factors</a>
                        <img class="title_link_icon" src="../../img/link_icon.png" alt="link">
                    </div>
                </h3>

                <h2 id="download">
                    <div class="title_container">
                        <a href="#download" class="title_link">Download</a>
                        <img class="title_link_icon" src="../../img/link_icon.png" alt="link">
                    </div>
                </h2>
                <ul class="h3_p">
                    <li>
                        We provide all the original sequence data and downstream analysis data generated from this
                        study. You can view detailed
                        descriptions of these files by clicking the help button on each download card. Both single and
                        multiple file downloads
                        are supported.
                    </li>
                </ul>

                <h2 id="about">
                    <div class="title_container">
                        <a href="#about" class="title_link">About</a>
                        <img class="title_link_icon" src="../../img/link_icon.png" alt="link">
                    </div>
                </h2>

                <h3 id="tutorial">
                    <div class="title_container">
                        <a href="#tutorial" class="title_link">Tutorial</a>
                        <img class="title_link_icon" src="../../img/link_icon.png" alt="link">
                    </div>
                </h3>
                <ul class="h3_p">
                    <li>The tutorial page you're currently viewing.</li>
                </ul>

                <h3 id="contact">
                    <div class="title_container">
                        <a href="#contact" class="title_link">Contact</a>
                        <img class="title_link_icon" src="../../img/link_icon.png" alt="link">
                    </div>
                </h3>
                <ul class="h3_p">
                    <li>
                        This section lists the contributors and supervisors of the project, along with their contact
                        information and addresses.
                        You can also find the source code for the project website here, and we welcome any feedback or
                        suggestions for
                        improvements to the site.
                    </li>
                </ul>

                <h2 id="qna">
                    <div class="title_container">
                        <a href="#qna" class="title_link">Q & A</a>
                        <img class="title_link_icon" src="../../img/link_icon.png" alt="link">
                    </div>
                </h2>
                <ul class="h3_p">
                    <li class="qna_question">Why is the website layout wider than usual? How can I adjust the browser's
                        width
                        settings?</li>
                    <li class="qna_answer">
                        Our website features rich visual elements designed to provide the best experience on screens
                        with a normal desktop
                        width. While the site is accessible on mobile devices, the browsing performance on mobile may
                        not deliver the optimal
                        visual experience. Therefore, we recommend using a computer display with a width greater than
                        1400px for the best
                        results visualization.

                        If the website appears too wide in your browser, you can adjust the browser's zoom settings to
                        resize the layout:
                        <ul>
                            <li>For Windows: Use <span class="code_span">"Ctrl" + "+"</span> to zoom in or <span
                                    class="code_span">"Ctrl" + "-"</span> to zoom out. You can also use
                                <span class="code_span">"Ctrl" + mouse wheel up/down</span>.
                            </li>
                            <li>For MacOS: Use <span class="code_span">"Command (⌘)" + "+"</span> to zoom in or <span
                                    class="code_span">"Command (⌘)" + "-"</span> to zoom out.
                                Alternatively, use <span class="code_span">"Command (⌘)" + mouse wheel up/down</span> .
                            </li>
                        </ul>
                    </li>
                </ul>





                <!-- 隐藏的放大图片容器 -->
                <div class="image_modal">
                    <div class="close_button">&times;</div>
                    <div class="image_modal_content">
                        <img src="#" alt="Zoomed Image" class="zoomed_image" title="Use mouse wheel to zoom in/out">
                        <div class="zoomed_caption">Fig 1. This is the title and description of the image</div>
                    </div>
                </div>

            </div>



            <!-- 导航列表 -->
            <div id="nav_list" class="nav_list">
                <ul>
                    <li><a href="#homepage" class="nav_link first_level_link active">Homepage</a>
                        <ul>
                            <li><a href="#abstract_citation" class="nav_link second_level_link">Abstract & Citation</a>
                            </li>
                            <li><a href="#statistics" class="nav_link second_level_link">Statistics</a></li>
                            <li><a href="#resources_list" class="nav_link second_level_link">Resources List</a></li>
                            <li><a href="#news_list" class="nav_link second_level_link">News List</a></li>
                        </ul>
                    </li>
                    <li><a href="#navigation_bar" class="nav_link first_level_link">Navigation Bar</a>
                        <ul>
                            <li><a href="#responsive_layout" class="nav_link second_level_link">Responsive Layout</a>
                            </li>
                            <li><a href="#global_search" class="nav_link second_level_link">Global Search</a></li>
                        </ul>
                    </li>
                    <!-- 继续添加其他一级和二级标题到导航列表 -->

                    <li><a href="#graph_summary" class="nav_link first_level_link">Graph Summary</a></li>
                    <li><a href="#data_resources" class="nav_link first_level_link">Data Resources</a>
                        <ul>
                            <li><a href="#full_length_transcriptome" class="nav_link second_level_link">Full-length
                                    Transcriptome</a></li>
                            <li><a href="#haplotype_SNP" class="nav_link second_level_link">Haplotype && SNP</a></li>
                            <li><a href="#gene_expression_profile" class="nav_link second_level_link">Gene Expression
                                    Profile</a>
                            </li>
                            <li><a href="#single_co_expression_network" class="nav_link second_level_link">Single
                                    Co-expression
                                    Network</a></li>
                            <li><a href="#hub_co_expression_network" class="nav_link second_level_link">Hub
                                    Co-expression
                                    Network</a></li>
                        </ul>
                    </li>

                    <li><a href="#tools" class="nav_link first_level_link">Tools</a>
                        <ul>
                            <li><a href="#blast" class="nav_link second_level_link">BLAST</a></li>
                            <li><a href="#annotation" class="nav_link second_level_link">Annotation</a></li>
                            <li><a href="#transcript_factors" class="nav_link second_level_link">Transcript Factors</a>
                            </li>
                        </ul>
                    </li>

                    <li><a href="#download" class="nav_link first_level_link">Download</a></li>
                    <li><a href="#about" class="nav_link first_level_link">About</a>
                        <ul>
                            <li><a href="#tutorial" class="nav_link second_level_link">Tutorial</a></li>
                            <li><a href="#contact" class="nav_link second_level_link">Contact</a></li>
                        </ul>
                    </li>
                    <li><a href="#qna" class="nav_link first_level_link">Q & A</a></li>

                </ul>
            </div>

            <!-- 导航列表图标 -->

            <button class="expand_list_button" id="expand_list_button">
                <img src="../../img/menu_icon.png" alt="menu">
            </button>


            <button class="click_to_close2" close-target-id="nav_list">
                <img src="../../img/close_icon.png" class="close_icon" alt="close">
            </button>




        </div>

        <div class="footer_container">
            <div class="logo_container">
                <div class="logo_box click_to_open_new_url" new-url="https://www.gxu.edu.cn/">
                    <img src="../../img/GXU_logo.png" alt="logo">
                </div>
                <div class="logo_box click_to_open_new_url" new-url="https://sklcusa.gxu.edu.cn/">
                    <img src="../../img/GuoZhong.png" alt="logo">
                </div>
                <div class="logo_box click_to_open_new_url" new-url="https://www.hzau.edu.cn/index.htm">
                    <img src="../../img/HZAU.png" alt="logo">
                </div>
                <div class="revolver_maps">
                    <script type='text/javascript' id='clustrmaps'
                        src='//cdn.clustrmaps.com/map_v2.js?cl=ffffff&w=a&t=n&d=aDEPs5hhknuFihFSlTig-Z5vc0AaI4FYpqSo6qOE14E&co=00411a'></script>
                </div>

            </div>

            <h1 class="development_info">Developed by the Chen LL’s Lab at GuangXi University and Xie WB’s Lab at
                HuaZhong
                Agricultural University</h1>

            <div class="copyright_revolvermap_container">
                <div class="copyright_container">
                    <h1 class="contact_info">Contact: <a class="email_address">1740569155@qq.com</a>, <a
                            class="email_address">zhaozhiyuan@webmail.hzau.edu.cn</a>
                    </h1>
                    <h1 class="copyright">Copyright © 2020-2030 All rights reserved</h1>
                </div>

            </div>

        </div>




        <!-- 全局提示框，触发showCustomAlert方法之后才显示（copy、download） -->
        <div id="customAlert" class="custom_alert"></div>

        <!-- 全局覆盖层, 用于禁用用户交互 -->
        <div id="global_overlay"
            style="display:none; position:fixed; top:0; left:0; width:100%; height:100%; background: rgba(0, 0, 0, 0); z-index: 1000;">
        </div>

        <script type="module" src="../../js/test2/basic.js"></script>
        <script type="module" src="../../js/test2/mainTutorial.js"></script>
    </div>


</body>

</html>